app/models/user.rb

```diff
@@ -38,6 +38,7 @@ def deleted
 
   validates :email, :encrypted_password, :authentication_token, absence: true, if: :built_in?
   schema_validations except: [:encrypted_password]
+  validates :time_zone, inclusion: { in: ActiveSupport::TimeZone.zones_map.keys }, allow_nil: true
```

Comment:
I'm aware. What about in the time between the timezone is removed and the user not updating his profile? What time zone will he get?